exercises/two-bucket/description.md

```diff
@@ -1,30 +1,34 @@
 # Description
 
-Given two buckets of different size, demonstrate how to measure an exact number of liters by strategically transferring liters of fluid between the buckets.
+Given two buckets of different size and which bucket to fill first, determine how many actions are required to measure an exact number of liters by strategically transferring fluid between the buckets. Since this mathematical problem is subject to a fair amount of interpretation and an individual's approach, the tests have been written specifically to expect one overarching solution.
 
-Since this mathematical problem is fairly subject to interpretation / individual approach, the tests have been written specifically to expect one overarching solution.
+There are some rules that your solution must follow:
 
-To help, the tests provide you with which bucket to fill first. That means, when starting with the larger bucket full, you are NOT allowed at any point to have the smaller bucket full and the larger bucket empty (aka, the opposite starting point); that would defeat the purpose of comparing both approaches!
+- You can only do one action at a time.
+- There are only 3 possible actions:
+  1. Pouring one bucket into the other bucket until:
```

Comment:
adding something like "either" here may help to clarify that it's not "both"

Reply:
Agree

Reply:
```suggestion
  1. Pouring one bucket into the other bucket until either:
```